profiler/profile.go

```diff
@@ -216,6 +229,108 @@ func goroutineProfile(cfg *config) (*profile, error) {
 	}, nil
 }
 
+func goroutineWaitProfile(cfg *config) (*profile, error) {
+	var (
+		text  = &bytes.Buffer{}
+		pprof = &bytes.Buffer{}
+		start = now()
+	)
+	if err := lookupProfile(GoroutineProfile.String(), text, 2); err != nil {
+		return nil, err
+	} else if err := goroutineDebug2ToPprof(text, pprof); err != nil {
+		return nil, err
+	}
+	end := now()
+	tags := append(cfg.tags, GoroutineWaitProfile.Tag())
+	cfg.statsd.Timing("datadog.profiler.go.collect_time", end.Sub(start), tags, 1)
```

Comment:
@pmbauer where does this data end up? In the users account? Our internal account? Also: Why are we using the same counter name for all the goroutine profiles? Would love to learn more about this.

Reply:
@felixge this is telemetry that ends up in the users account.  Metric systems like graphite have to distinguish between time series with unique metric names, but others like Datadog's supports tags.  Metric time series are unique by metric name + tag set.

Reply:
this is telemetry that ends up in the users account. Metric systems like graphite have to distinguish between time series with unique metric names, but others like Datadog's supports tags. Metric time series are unique by metric name + tag set.

Reply:
Thanks, got a follow-up Q about this, will ping you in slack.